tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -408,62 +400,49 @@ def predict(
+                `patch` mode, the input must be either a list of images,
+                a list of image file paths or a numpy array of an image
+                list. When using `tile` or `wsi` mode, the input must
+                be a list of file paths.
+            masks (list): List of masks. Only utilised when processing image
+                tiles and whole-slide images. Patches are only processed
+                if they are witin a masked area. If not provided, then a
+                tissue mask will be automatically generated for whole-slide
+                images or the entire image is processed for image tiles.
             labels: List of labels. If using `tile` or `wsi` mode, then only a
             single label per image tile or whole-slide image is supported.
             mode (str): Type of input to process. Choose from either `patch`, `tile` or
                 `wsi`.
-
             return_probabilities (bool): Whether to return per-class probabilities.
-
             return_labels (bool): Whether to return the labels with the predictions.
-
             on_gpu (bool): whether to run model on the GPU.
-
             patch_size (tuple): Size of patches input to the model. Patches are at
                 requested read resolution, not with respect to level 0, and must be
                 positive.
-
             stride_size (tuple): Stride using during tile and WSI processing. Stride
                 is at requested read resolution, not with respect to to level 0, and
                 must be positive. If not provided, `stride_size=patch_size`.
-
             resolution (float): Resolution used for reading the image.
-
             units (str): Units of resolution used for reading the image. Choose from
                 either `level`, `power` or `mpp`.
-
             merge_predictions (bool): Whether to merge the predictions to form
             a 2-dimensional map. This is only applicable for `mode='wsi'` or
             `mode='tile'`.
-
             save_dir (str): Output directory when processing multiple tiles and
                 whole-slide images. By default, it is folder `output` where the
                 running script is invoked.
-
         Returns:
             output (ndarray, dict): Model predictions of the input dataset.
                 If multiple image tiles or whole-slide images are provided as input,
                 then results are saved to `save_dir` and a dictionary indicating save
                 location for each input is return.
-
                 The dict has following format:
                 - img_path: path of the input image.
                     - raw: path to save location for raw prediction, saved in .json.
                     - merged: path to .npy contain merged predictions if
                     `merge_predictions` is `True`.
-
-                For example
+                Example:
                 >>> wsis = ['wsi1.svs', 'wsi2.svs']
```

Comment:
Please check the spacing and make it consistent. Please see https://github.com/TIA-Lab/tiatoolbox/blob/develop/tiatoolbox/wsicore/save_tiles.py as an example or the notebook.